lib/node_modules/@stdlib/math/base/complex/divide/benchmark/benchmark.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var randu = require( '@stdlib/math/base/random/randu' );
+var isArray = require( '@stdlib/assert/is-array' );
+var pkg = require( './../package.json' ).name;
+var cdiv = require( './../lib' );
+
+
+// MAIN //
+
+bench( pkg, function benchmark( b ) {
+	var re1;
+	var re2;
+	var im1;
+	var im2;
+	var y;
+	var i;
+
+	b.tic();
+	for ( i = 0; i < b.iterations; i++ ) {
+		re1 = ( randu()*1000.0 ) - 500.0;
+		im1 = ( randu()*1000.0 ) - 500.0;
+		re2 = ( randu()*1000.0 ) - 500.0;
+		im2 = ( randu()*1000.0 ) - 500.0;
```

Comment:
Yes, it is likely that `randu` is rate limiting here. Benchmarks, however, are not necessarily about absolute performance. While we try to minimize operations which may obscure implementation performance, we are primarily concerned with use benchmarking as a guide for identifying problematic implementations (e.g., perf is no t in accordance with expectation), making relative comparisons to reference implementations (e.g., provided equivalent benchmarks in two langs, what is the relative difference? Here, if they both have 4 calls to a PRNG, then we can reasonably draw perf conclusions.), and protecting against future perf regressions.

If you are truly worried about the 4 calls, then you can just update only one of the values and hold the other 3 fixed (e.g., move them to above the `for` loop). At least one of the values needs to be overwritten in order to insure that the compiler does not perform LICM optimizations, removing the `for` loop altogether. If commonly do a similar protection when benchmarking functions which operate over arrays, where we only update one element in the input array to ensure that a different output is always returned by the tested function.